app/_src/gateway/plugin-development/wasm.md

```diff
@@ -0,0 +1,165 @@
+---
+title: WebAssembly
+book: plugin_dev
+chapter: 11
+---
+
+# Hello World: Your First Proxy-Wasm Filter
+
+Filters can be written in almost any language that supports WebAssembly as a
+compilation target, but the easiest  and most ergonomic way to get started is by
+using a language which has an existing [Proxy-Wasm](https://github.com/proxy-wasm/spec)
+SDK, such as [Go](https://github.com/tetratelabs/proxy-wasm-go-sdk/),
+[C++](https://github.com/proxy-wasm/proxy-wasm-cpp-sdk/), or [Rust](https://github.com/proxy-wasm/proxy-wasm-rust-sdk/).
+
+This section describes how to create and use a basic Proxy-Wasm filter to
+terminate the request after sending a custom response.
+
+## The Code
```

Comment:
Instead of having code and build instructions in this page, I think we should point people to our starter repositories https://github.com/Kong/proxy-wasm-rust-filter-template/ and https://github.com/Kong/proxy-wasm-go-filter-template/ — this way we can update the repos for any future changes, and the rest of this page works the same, whether one's building a filter in Rust or Go.

Reply:
Removed the "The Code" and "Building" sections. What do you think about the document as it is?